class-jetpack-currencies.php

```diff
@@ -0,0 +1,180 @@
+<?php
```

Comment:
Maybe moving this to its own class would be a good opportunity to add some tests as well?

Reply:
Good call, added in 459a9f5.